Gruntfile.js

```diff
@@ -229,10 +231,9 @@ module.exports = function(grunt) {
   grunt.registerTask('default', ['browserify']);
 
   grunt.registerTask('test', [
-    'jscs',
-    'jshint',
+    'eslint',
     process.env.JENKINS_HOME ? 'mochaTest:unit-xml' : 'mochaTest:unit',
-   'karma:unit-once']);
+    'karma:unit-once']);
```

Comment:
was confused because we didn't break out the last ]) to another line. ie

```  
grunt.registerTask('test', [
  'eslint',
  process.env.JENKINS_HOME ? 'mochaTest:unit-xml' : 'mochaTest:unit',
  'karma:unit-once']);
  'karma:unit-once'
]);
```